test/e2e/common/utils.py

```diff
@@ -154,8 +150,7 @@ def explain_response(service_name, input_json):
     )
     # temporary sleep until this is fixed https://github.com/kserve/kserve/issues/604
     time.sleep(10)
```

Comment:
https://github.com/kserve/kserve/issues/604 has been merged, can we remove this or are we waiting on something else?

Reply:
IDK. Probably, there is no need to way.
We should contribute such change directly to upstream.